addon/globalPlugins/EnhancedFindDialog/guiHelper.py

```diff
@@ -111,31 +156,56 @@ def __init__(self, parent, cursorManager, profile, searchEntries, reverseSearch)
 		# present the last searched term  selected by default
 		if searchEntries:
 			self.findTextField.Select(SEARCH_HISTORY_MOST_RECENT_INDEX)
-
+		searchTypeHelper = guiHelper.BoxSizerHelper(
+			self, orientation=wx.HORIZONTAL)
+		self._searchTypeCtrl = searchTypeHelper.addItem(wx.RadioBox(
+			self,
+			# Translators: A radio box to select the search type.
+			label=__("Search type"), choices=getSearchTypes(),
```

Comment:
Changed the following:
* remplace double underscore by single one since the string is not in NVDA basis
* Added colon at the end, like in new profile dialog, because look nicer and more correlated with the two radio button below; just personal opinion though for this second point so feel free to ignore  it.

```suggestion
			label=_("Search type:"), choices=getSearchTypes(),
```